### PR TITLE
Make VaadinService.dependencyFilters unmodifiable

### PR DESCRIPTION
**Pull request by fluorumlabs**
### Warning: Exposed mutable collection ([CERT](https://wiki.sei.cmu.edu/confluence/display/java/OBJ05-J.+Do+not+return+references+to+private+mutable+class+members))

Returning references to internal mutable members of a class can compromise an application's security, both by breaking encapsulation and by providing the opportunity to corrupt the internal state of the class (whether accidentally or maliciously). As a result, programs must not return references to private mutable classes.

Class: **`com.vaadin.flow.server.VaadinService`**
Scope: **singleton**
Field: **`private Iterable<DependencyFilter> VaadinService.dependencyFilters`**
Value: `(ArrayList<E>) []`
Value is read:
- **`public Iterable<DependencyFilter> VaadinService.getDependencyFilters()`**
Value is written:
- **`private void VaadinService.lambda$init$1(ServiceInitEvent, List<E>)`**
Value is returned:
- **`public Iterable<DependencyFilter> VaadinService.getDependencyFilters()`**

### CODE REVIEW

File: flow-server/src/test/java/com/vaadin/flow/server/BootstrapHandlerTest.java [hunk 1]
```diff
@@ -6,12 +6,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
-import java.util.Collections;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Locale;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```
Comment (pleku) [inline flow-server/src/test/java/com/vaadin/flow/server/BootstrapHandlerTest.java:9]: No `*` imports according to the coding conventions (just noticed that the conventions are not up to date so we need to fix that immediately)

**Review by pleku — CHANGES_REQUESTED** [review]
Only one import formatting change please

File: flow-server/src/test/java/com/vaadin/flow/server/BootstrapHandlerTest.java (hunk 1 above)
Comment (fluorumlabs) [reply flow-server/src/test/java/com/vaadin/flow/server/BootstrapHandlerTest.java]: Fixed

**Review by pleku — DISMISSED** [review]

**Review by pleku — APPROVED** [review]

**Comment by mshabarov** [comment]
Cherry picked to `2.5`

**Pull request merged**